web/text_layer_builder.js

```diff
@@ -53,6 +54,17 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
   }
 
   TextLayerBuilder.prototype = {
+    _afterRender: function TextLayerBuilder_afterRender() {
+      if (!this.renderingDone) {
```

Comment:
Fixed; I went for the `_finishRendering` option since that meant less code duplication.